report_breach_level_DE_warning.py

```diff
@@ -0,0 +1,11 @@
+
+

```

Comment:
This file is very similar to `report_breach_level_EN_warning.py`.
Same for the other EN and DE files.
Issue: For example, if the tolerance needs to change, it's possible to change in EN and forget in DE - if we focus testing in English.
To prevent this, we need to repeat ALL tests for DE and EN.
Actually the only thing that changes is the message. Can we separate the messages alone?

Reply:
I have removed the duplication by introducing constants for language.

![image](https://user-images.githubusercontent.com/13776900/111633309-5d8dca00-881b-11eb-9f79-135f34aac37d.png)